README.md

```diff
@@ -1,4 +1,7 @@
-# Stripe checkout for Ember [![Build Status](https://travis-ci.org/sweettooth/ember-cli-stripe.svg?branch=master)](http://travis-ci.org/sweettooth/ember-cli-stripe)
+ember-cli-stripe
+==============================================================================
+
+# Stripe checkout for Ember [![Build Status](https://travis-ci.org/smile-io/ember-cli-stripe.svg?branch=master)](http://travis-ci.org/smile-io/ember-cli-stripe)
```

Comment:
keeping it like this we basically have to H1's
![Screen Shot 2019-07-15 at 3 33 22 PM](https://user-images.githubusercontent.com/160955/61216405-f2225980-a715-11e9-9d85-da6af3ad58ab.png)